openshift/mongo.helm.yml

```diff
@@ -0,0 +1,21 @@
+fullnameOverride: hcap-mongo
+persistence:
+  size: 4Gi
+
+podSecurityContext:
+  enabled: false
+
+containerSecurityContext:
+  enabled: false
+
+arbiter:
+  podSecurityContext:
+    enabled: false
+
+architecture: replicaset
+image:
+  registry: "public.ecr.aws"
+
+auth:
+  username: hcap
```

Comment:
This should be connected to any secret. This is a public repository, this is a big security issue. or at-least connect these auth attribute with input parameter